test/modules/scope.spec.js

```diff
@@ -167,7 +167,10 @@ describe('modules/scope', () => {
     });
 
     it('should have focused the selector', () => {
-      expect(wrapper.find('.primary').instance() === document.activeElement).toBeTrue();
```

Comment:
The test wasn't breaking so is this change valid at all?

Reply:
The test was breaking with adding any form of delay with focusing. The test environment didn't have the same issue as browsers and was able to apply focus to the DOM element immediately. Since now there is a very slight delay in adding focus, I set a wait to 100 milliseconds to get it to fire.